dvc/command/experiments/show.py

```diff
@@ -604,14 +605,24 @@ def add_parser(experiments_subparsers, parent_parser):
         default=False,
         help="Show experiments derived from all Git commits.",
     )
+    experiments_show_parser.add_argument(
+        "--rev",
+        type=str,
+        default=None,
+        help=(
+            "Show experiments derived from the specified revision. "
+            "Defaults to HEAD if none of `--rev`,`-a`,`-A`,`-T` is specified."
+        ),
+        metavar="<rev>",
+    )
     experiments_show_parser.add_argument(
         "-n",
         "--num",
         type=int,
         default=1,
         dest="num",
         metavar="<num>",
-        help="Show the last `num` commits from HEAD.",
+        help="Show the last `num` commits from <rev>.",
```

Comment:
OK so I know this is already part of the codebase, sorry. But should the metavar be <int>? "num" is not informative since the option has the same name. "int" is a type hint at least, which some time ago we agreed to use for metavars AFAIR. Either way it should be pretty obvious here but suggesting as a UX improvement. Cc @dberenbaum

Reply:
Moved to https://github.com/iterative/dvc/issues/7314 actually.